src/ServiceFabricPubSub/SubscriberService/packages.config

```diff
@@ -1,8 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's being refactored to a .NET core app, the project references are in the csproj.